xonsh/environ.py

```diff
@@ -388,7 +388,8 @@ def xonshconfig(env):
     'PROMPT': VarDocs(
         'The prompt text. May contain keyword arguments which are '
         "auto-formatted, see 'Customizing the Prompt' at "
-        'http://xon.sh/tutorial.html#customizing-the-prompt.',
+        'http://xon.sh/tutorial.html#customizing-the-prompt. '
+        'This value is never inherited from parent processes.'),
```

Comment:
Test failure here, @adqm -- I think you want to remove the paren at the end of this line.